From 4710568c7c936343fad849e07edb5a15c2d44af4 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Fri, 16 Aug 2019 15:49:39 +0800
Subject: [PATCH 1/3] suppress warnings being marked as errors

Otherwise when building with clang-8:

| clang-8: error: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated [-Werror,-Wdeprecated]

Also see:
https://github.com/intel/intel-graphics-compiler/issues/106

Upstream-Status: Inappropriate
Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 IGC/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/IGC/CMakeLists.txt b/IGC/CMakeLists.txt
index f8b5e3f4..58cc4f37 100644
--- a/IGC/CMakeLists.txt
+++ b/IGC/CMakeLists.txt
@@ -2771,7 +2771,7 @@ foreach(_compilerFlagName IN ITEMS "CMAKE_CXX_FLAGS" "CMAKE_C_FLAGS")
           -march=corei7
           -mstackrealign
           -fms-extensions
-          -Werror
+          -Wno-error
           -Wno-unused-parameter
           -Wno-missing-field-initializers
           -Wwrite-strings
-- 
2.20.1

